### PR TITLE
312 fix images on frontend

### PR DESCRIPTION
**Pull request by dorianjagusch**
## Changes: 
- Add images to frontend
- Fix deleted user handling on frontend
- start of fixing history component (it still overflows tho)

### CODE REVIEW

**Review by azarSarikhani — APPROVED** [review]

**Review by sakarisalmi — APPROVED** [review]

File: frontend/static/js/services/ARequestService.js [hunk 1]
```diff
@@ -138,7 +138,7 @@ class ARequestService {
 			method: 'DELETE',
 			headers: {
 				'X-CSRFToken': getCookie('csrftoken'),
-				'Content-Type': 'application/json',
```
Comment (merituulie) [inline frontend/static/js/services/ARequestService.js:141]: I dont think this is a good idea, the rest of the deletions will be json wont they?

Comment (merituulie) [reply frontend/static/js/services/ARequestService.js:141]: In general, deleteRequest doesnt really take a body, just an id of the resource we want to delete. how about an overload?

File: frontend/static/js/services/statsService.js [hunk 2]
```diff
@@ -11,15 +11,23 @@ export default class StatsService extends ARequestService {
...
+				},
+				credentials: 'include',
+			});
+			if (!response.ok) {
+				throw new Error('Network response was not ok');
+			}
+			const svgText = await response.text();
+			return svgText;
+		} catch (error) {
+			console.error("Couldn't fetch data", error);
```
Comment (merituulie) [inline frontend/static/js/services/statsService.js:30]: notify?

File: frontend/static/js/views/dashboardView.js [hunk 3]
```diff
@@ -64,15 +53,14 @@ export default class extends AView {
 				return {
 					match_id: match.id,
 					opponent: opponent.username,
-					opponentId: opponent.id,
```
Comment (merituulie) [inline frontend/static/js/views/dashboardView.js:67]: fetch and pull!

**Review by azarSarikhani — APPROVED** [review]

**Pull request merged**